pandas/core/arrays/base.py

```diff
@@ -32,6 +32,18 @@
 _extension_array_shared_docs = dict()  # type: Dict[str, str]
 
 
+def safe_ea_cast(cls, obj, dtype=None):
```

Comment:
Wondering if something like `try_cast_to_ea` would be clearer (or with ea in full), as it doesn't necessarily give a ea as result (the "safe_ea_cast" doesn't really imply that)

Reply:
sure